templates/managed/index.html

```diff
@@ -134,455 +150,402 @@ <h1>Managed IT Services</h1>
         </div>
       </div>
     </div>
-    <p>
-      <a href="/engage/private-cloud-cost-analysis-webinar">Watch "Hosted private cloud infrastructure: A cost analysis" webinar</a>
-    </p>
   </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-8">
-      <h2>
-        A frictionless cloud journey
-      </h2>
-      <p>
-        Building your private/hybrid cloud puts you at the centre of innovation. Whether you're transitioning to the cloud or already running a fully operational one, it is important to structure a business-centric IT that helps your organisation focus on innovation, cut down costs and realise the benefits of the cloud.
-      </p>
-      <p>
-        A recent Gartner <a href="https://www.gartner.com/en/publications/cio-top-priorities-leadership-vision-2021">report</a> shows that the biggest infrastructure and operations challenges facing CIOs are insufficient skilled resources and managing the technical debt. Our managed services allow you to focus on your business strategy, fill any skill gaps and offload the technical complexity of cloud operations.
-      </p>
-    </div>
+<section class="u-fixed-width">
+  <hr class="p-rule" />
+  <div class="p-strip is-deep">
+    <h2>
+      Case study: <a href="https://ubuntu.com/engage/kubernetes-openstack-case-study-firmus">Firmus builds a sustainable
+        supercloud with&nbsp;Canonical</a>
+    </h2>
   </div>
 </section>
 
-<section class="p-strip">
-  <div class="row">
-    <div class="col-8">
-      <h2>Human Resources Management</h2>
-      <p>Increase labor efficiency through access to qualified, experienced cloud engineers for your operations. Leverage your IT engineers’ capabilities to prepare your business for the future.</p>
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>We mind your ops,<br />you mind your business</h2>
     </div>
-    <div class="col-4 u-hide--medium u-hide--small u-align--center">
-      {{
-      image (
-      url="https://assets.ubuntu.com/v1/7cacf0d8-little+men.svg",
-      alt="",
-      width="185",
-      height="145",
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
+    <div class="col">
+      <p>Open source IT solutions are the future when it comes to technology, but can often be challenging to maintain.
+        Managed IT services are a solution that allows you to harness the power of open source without the hassle of
+        maintenance.
+      </p>
+      <p>
+        Canonical’s Managed Solutions is a service that identifies, deploys, and manages open source ecosystems for you
+        at predictable costs. This way, you can access our market-leading open source expertise and focus more of your
+        resources on your core business.
+      </p>
+      <hr />
+      <a class="p-button--positive js-invoke-modal" href="/managed#get-in-touch">Tell us what you need</a>
```

Comment:
The href should go to a static contract form as a fall back for no-js

Reply:
Good point, thanks 👍